debian/libostree-1-1.symbols

```diff
@@ -150,6 +150,7 @@ libostree-1.so.1 libostree-1-1 #MINVER#
  ostree_remote_unref@LIBOSTREE_2017.6_EXPERIMENTAL 2017.8
  ostree_remote_get_name@LIBOSTREE_2017.7_EXPERIMENTAL 2017.8
  ostree_remote_get_type@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless3
+ ostree_remote_get_url@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless5
```

Comment:
I think you also need a line up at the top of the file (like the others there):
```
LIBOSTREE_2017.14_EXPERIMENTAL@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless5
```

I wish I understood this file format properly.

Reply:
With that addition, please feel free to merge/push both branches.

Reply:
The global `LIBOSTREE_2017.14_EXPERIMENTAL@LIBOSTREE_2017.14_EXPERIMENTAL` is already there at the top from when you added it for `0endless3`.

Reply:
See dpkg-gensymbols(1) and deb-symbols(5) for the format of this file.

ostree is a little unusual since it uses symbol versions (the `@LIBOSTREE_$VERSION` annotations) while most libraries don't. I guess then there's also an entry for the symbol version itself (e.g., `LIBOSTREE_$VERSION@LIBOSTREE_$VERSION`). Either way, each line is basically just the symbol name and what package version it was added in.

Reply:
In conclusion: I’m wrong. The PR is fine as it stands. Merging!